kotlin-eclipse-core/src/org/jetbrains/kotlin/core/preferences/KotlinProperties.kt

```diff
@@ -0,0 +1,12 @@
+package org.jetbrains.kotlin.core.preferences
+
+import org.jetbrains.kotlin.core.Activator
+import org.jetbrains.kotlin.config.JvmTarget
+import org.eclipse.core.runtime.preferences.IScopeContext
+import org.eclipse.core.runtime.preferences.DefaultScope
+import org.eclipse.core.resources.IProject
+import org.eclipse.core.resources.ProjectScope
+
+class KotlinProperties(scope: IScopeContext = DefaultScope.INSTANCE) : Preferences(Activator.PLUGIN_ID, scope) {
+	var jvmTarget by EnumPreference<JvmTarget>()
```

Comment:
It cannot be `val`, because setting this property is the only way to change jvm target version. It is used as mutable property in `KotlinCompilerPropertyPage`